src/app/gui/map/mapservice.js

```diff
@@ -1341,9 +1341,12 @@ proto._updateMapControlsLayout = function({width, height}={}) {
         }
         if (changedAndMoreSpace.changed) {
           const mapControslHeight = this.state.mapControl.grid[this.state.mapControl.currentIndex].columns * this.state.mapcontrolSizes.minWidth;
-          const mapControlsWidth = this.state.mapControl.grid[this.state.mapControl.currentIndex].rows * this.state.mapcontrolSizes.minWidth;
           this.state.mapcontrolDOM.css('height', `${mapControslHeight}px`);
-          this.state.mapcontrolDOM.css('width', `${mapControlsWidth}px`);
+          /**
+           *  @deprecate since v3.9.0.
+           */
+          //const mapControlsWidth = this.state.mapControl.grid[this.state.mapControl.currentIndex].rows * this.state.mapcontrolSizes.minWidth;
+          //this.state.mapcontrolDOM.css('width', `${mapControlsWidth}px`);
```

Comment:
@volterra79 not so easy to understand, what has been deprecated?

Reply:
@Raruto because display:flex property do it automatically